Source/PermissionTypes/UserNotifications.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  UserNotifications.swift
+//  Permission
+//
+//  Created by Andreas Grauel on 20.01.17.
+//  Copyright © 2017 delba. All rights reserved.
+//
+
+#if PERMISSION_USERNOTIFICATIONS
+import UserNotifications
+
+internal extension Permission {
+    var statusUserNotifications: PermissionStatus {
```

Comment:
Could we keep the structure of the `Motion` permission (that also uses "synchronous" calls) please ?